### PR TITLE
Add information for assert and for change notices

### PR DESCRIPTION
**Pull request by bertrand-lorentz**

### CODE REVIEW

**Review by rouschr — APPROVED** [review]
The commits / diff are crystal clear so it is OK for me.

**Review by rouschr — DISMISSED** [review]

File: schematrons/dynamic/entry.sch [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8" ?>
```
Comment (rousso) [inline schematrons/dynamic/entry.sch:1]: We forgot to rename this file. 

We have to options: Either accept it as is, or make (as I would have preferred):

1. rename **entry.sch** to **complete-validation.sch** 
2. and the different stage files to validation-stage-1a.sch, validation-stage-1b.sch, validation-stage-2.sch an so on...

**Review by rousso — COMMENTED** [review]
Apart form schematron filenames everything looks good to me

File: schematrons/dynamic/entry.sch (hunk 1 above)
Comment (rouschr) [reply schematrons/dynamic/entry.sch:1]: or just validation-all.sch

Comment (bertrand-lorentz) [reply schematrons/dynamic/entry.sch:1]: I've added a commit to rename the schematron files.

**Review by rouschr — APPROVED** [review]

**Pull request merged**